crates/matrix-sdk/src/client/builder.rs

```diff
@@ -63,7 +66,7 @@ pub struct ClientBuilder {
     request_config: RequestConfig,
     respect_login_well_known: bool,
     appservice_mode: bool,
-    server_versions: Option<Arc<[MatrixVersion]>>,
+    server_versions: Option<Vec<MatrixVersion>>,
```

Comment:
Switching from `Arc<[_]>` to `Vec<_>` and using a borrow in `HttpClient` makes the future returned by `Client::send` not `'static`, can you please revert that part?

Reply:
Also for `Client::server_versions`?